### DISCUSSION TITLE
Anregung: Filter "Niemals in Deutschland gezeigt" entfernen für Feuerwehr-Hydranten

### DISCUSSION THREAD

**Discussion by gitmuti** [Helium314/SCEE#618] [Ideas]
Es gibt die Aufgabe "Was ist die Referenznummer hier?" für Feuerwehr-Hydranten.
Diese Aufgabe wird "Niemals in Deutschland" gezeigt.
Bereits seit einigen Jahren erhalten die Hydranten-Schilder in Deutschland auch eine Referenznummer. Siehe Bild rechts oben in [https://de.wikipedia.org/wiki/Hinweisschilder_zu_Stra%C3%9Feneinbauten](https://de.wikipedia.org/wiki/Hinweisschilder_zu_Stra%C3%9Feneinbauten) 
bzw. der Link zum Bild direkt ist [https://de.wikipedia.org/wiki/Hinweisschilder_zu_Stra%C3%9Feneinbauten#/media/Datei:Hinweisschilder_f%C3%BCr_Hydrant_Wasser_und_Gas_102_2002.jpg ](https://de.wikipedia.org/wiki/Hinweisschilder_zu_Stra%C3%9Feneinbauten#/media/Datei:Hinweisschilder_f%C3%BCr_Hydrant_Wasser_und_Gas_102_2002.jpg )
Hier in diesem Beispiel ist die Referenznummer 6846.
Ich rege an, den Filter "Niemals in Deutschland gezeigt" für Hydranten zu entfernen.

**Comment by mcliquid** [comment]
Hallo, kennst du die früheren Diskussionen zu diesem Thema?
Wenn ja, was hat sich im Vergleich zum letzten Jahr in der Praxis geändert?
https://github.com/streetcomplete/StreetComplete/issues/3059
https://github.com/streetcomplete/StreetComplete/issues/5272
https://github.com/streetcomplete/StreetComplete/pull/4440

**Reply by HolgerJeromin** [reply]
I just asked a firefighter in Germany. He answered:

> This depends on the city or municipality. In my city they are not written on the signs.

**Reply by mcliquid** [reply]
> I just asked a firefighter in Germany. He answered:
> 
> > This depends on the city or municipality. In my city they are not written on the signs.

Thank you, I would have been surprised if the documents of our federal fire school were so different.
So my comment at the time should still be valid for large parts of Germany: https://github.com/streetcomplete/StreetComplete/issues/5272#issuecomment-1734518429

The fact is that (volunteer) fire brigades in Germany are organized from the individual federal states (e.g. Baden-Württemberg = [Landesfeuerwehrverband Baden-Württemberg](https://www.fwvbw.de/)). The training is organised via the [Landesfeuerwehrschule Baden-Württemberg](https://www.lfs-bw.de/). 

The equipment is maintained and financed by the local municipality. Only a few vehicles and items of equipment are provided by the district (Landkreis). The hydrants are placed by the responsible municipality. They can decide in co-operation with the local fire brigade whether it makes sense to reference the hydrants.  In our district, only one of 23 municipalities has decided to implement this.

**Comment by gitmuti** [comment]
Hallo, 
Danke für den Hinweis. Ich hatte das nicht gelesen, weil ich nur vorher das SCEE forum durchsucht hatte und nicht das normale StreetComplete forum. 
Ich habe sie jetzt gelesen. 
Ob sich was in der Praxis geändert hat, weiß ich nicht. 
Mir ist nur aufgefallen, dass es eine weitere Aufgabe gibt, wo die Referenznummer gefragt wird, und dort wird diese Frage auch gezeigt z.B. wenn railway=subway_entrance. Hier habe z.B. noch nie eine ID für einen U-Bahn-Eingang tatsächlich gefunden. 
Ich denke, es gibt halt Betreiber von Einrichtungen wie z.B. Hydranten, U-Bahn-Eingängen usw., die bringen eine ID an und andere nicht. Für mich wäre das kein Hintergrund, die Frage auszuschließen. 
In den Einstellungen könnte man ja die Frage enablen oder nicht enablen.
Wie gesagt, es ist nur eine Anregung.

**Reply by mcliquid** [reply]
> Ich denke, es gibt halt Betreiber von Einrichtungen wie z.B. Hydranten, U-Bahn-Eingängen usw., die bringen eine ID an und andere nicht.

Genau das war meine Aussage damals und eben auch nochmal als Antwort auf HolgerJeromin in Englisch: Ob ein Hydrant eine Referenznummer erhält, entscheidet einzig und allein die Gemeinde / Stadt, in der der Hydrant aufgestellt ist. Das kann regional sehr unterschiedlich sein, aber zumindest in Süddeutschland haben die allerwenigsten Kommunen eine Referenzierung / Nummerierung eingeführt. Aus diesem Grund wurde seinerzeit entschieden, die Abfrage in Deutschland und Österreich auszublenden, da sie sonst zu "spammy" wäre.

Da wir uns hier im SCEE-Repo befinden, ein kleiner Tipp von mir: Nimm doch eine der anderen `ref`-Quests und bearbeite die Abfrage dort. Zum Beispiel in dem Quest "Was ist die Referenznummer hier" mit dem grünen Plus (Rettungspunkt) Icon. 
Dort hast du als Abfrage:
```
nodes with
(
  highway = emergency_access_point or emergency = access_point
)
  and !name and !ref and noref != yes
  and ref:signed != no and !~"ref:.*"
```

Ergänze dort doch einfach eine Zeile mit `emergency = fire_hydrant` und du hast deinen eigenen Quest dafür aktiviert.

In Summe würde das dann so aussehen:
```
nodes with
(
  highway = emergency_access_point or emergency = access_point
  or emergency = fire_hydrant
)
  and !name and !ref and noref != yes
  and ref:signed != no and !~"ref:.*"
```

**Reply by mcliquid** [reply]
Noch ergänzend: Der `FireHydrantRef`-Quest ist momentan ausschließlich in der Schweiz und in Frankreich aktiviert. Ansonsten in keinem anderem Land.

**Reply by FloEdelmann** [reply]
Ich würde empfehlen, nicht den Quest für die Referenznummer von Rettungspunkten zu bearbeiten, sondern den Quest für die Referenznummer verschiedener Objekte (graues Symbol mit weißem Schlüsselanhänger). Sonst stimmt nämlich der Changeset-Kommentar („Survey access point refs“) nicht mit dem tatsächlichen Inhalt des Changesets überein.

**Comment by gitmuti** [comment]
OK. Danke für die Antworten und den Hinweis! Ich werde die Zeile mit emergency = fire_hydrant ergänzen.

**Comment by FloEdelmann** [comment]
See also https://github.com/Helium314/SCEE/issues/531#issuecomment-2039566642 where I suggested to lift the country restriction in SCEE.

**Reply by FloEdelmann** [reply]
I've created a new issue for this:

* https://github.com/Helium314/SCEE/issues/619
